### PR TITLE
Error message displayed when empty feedback form submitted.

### LINKED ISSUE #2960: No error message displayed when try to submit an empty feedback form.
**Actual Behaviour**

No error message is displayed when you try to submit an empty feedback form for a skill while editing.

**Expected Behaviour**

It should display an error message.

**Steps to reproduce it**

For reference visit https://susi.ai/Communication/EnglishAIMLL/en and add feedback & then edit your comment. While submitting an empty form, no error message will be displayed. 

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes, I would like to work on this.

### PR DESCRIPTION
**Pull request by plxity**
Fixes #2960 

Changes: 
1) Converted EditFeedback component to class base component.
2) Changed the way edit form was getting submitted without showing an error.

Demo Link: https://pr-2992-fossasia-susi-web-chat.surge.sh

Screenshot from 2019-10-28 14-42-22

### CODE REVIEW

**Comment by mariobehling** [comment]
Please provide a screenshot.

**Comment by plxity** [comment]
@mariobehling Please review it.

**Comment by akshatnitd** [comment]
It is duplicate of #2956. So please close the PR.

**Comment by plxity** [comment]
@akshatnitd SIr, this PR solves a different issue. It shows an error when an empty feedback form is displayed.

**Comment by plxity** [comment]
@Akshat-Jain Apologies for the delay. I will make the necessary changes and fix the issue.

**Comment by plxity** [comment]
@akshatnitd SIr, can I use class-based component for solving this?

**Comment by plxity** [comment]
`  const handleEditFeedbac = () => {
    let feedbackText = document.getElementById('edit-feedback').value;

    if (!feedbackText) {
      document.getElementById('edit-button').disabled = true;
    }
  };`

@akshatnitd Sir, in `onChange` event of input field I am passing this function. But the button is getting disabled. It changes the disabled property( I checked in the console) but as the UI is not getting rendered again.
Can you suggest an approach?

File: src/components/cms/SkillFeedbackPage/EditFeedbackDialog.js [hunk 1]
```diff
@@ -8,56 +8,64 @@ import FormControl from '@material-ui/core/FormControl';
...
+  render() {
+    const { handleClose, feedback, handleEditFeedback } = this.props;
+    const { errorText } = this.state;
+    return (
+      <React.Fragment>
+        <DialogTitle>Edit Feedback</DialogTitle>
+        <DialogContent>
+          <FormControl fullWidth={true}>
+            <OutlinedTextField
+              id="edit-feedback"
```
Comment (akshatnitd) [inline src/components/cms/SkillFeedbackPage/EditFeedbackDialog.js]: Remove this logic of getting value by `id`. Instead declare a state variable for it.

File: src/components/cms/SkillFeedbackPage/EditFeedbackDialog.js [hunk 2]
```diff
@@ -8,56 +8,64 @@ import FormControl from '@material-ui/core/FormControl';
...
+            variant="contained"
+            color="primary"
+            onClick={handleClose}
+          >
+            Cancel
+          </Button>
+          <Button
+            key={1}
+            variant="contained"
+            color="primary"
```
Comment (akshatnitd) [inline src/components/cms/SkillFeedbackPage/EditFeedbackDialog.js]: Here a prop `disabled` can be passed if `feedback.trim().length === 0`

**Pull request closed without merging**